src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes           from 'prop-types';
+import React               from 'react';
+import classNames          from 'classnames';
+import { Icon }            from 'Assets/Common';
+import { IconDanger }      from 'Assets/Common/icon-danger.jsx';
+import { IconInformation } from 'Assets/Common/icon-information.jsx';
+import { IconWarning }     from 'Assets/Common/icon-warning.jsx';
+import { IconSuccess }     from 'Assets/Common/icon-success.jsx';
+import CloseButton         from './close-button.jsx';
+import {
+    default_delay,
+    types }             from './constants';
+
+const Notification = ({
+    data,
+    removeNotification,
+}) => {
+    const destroy = (is_closed_by_user) => {
+        removeNotification(data);
+
+        if (typeof data.closeOnClick === 'function') {
```

Comment:
is there any case that data.closeOnClick is not a function. is this checking if data.closeOnClick is exists?